File: Java/TP3-4/src/Root.java
public class Root extends Usuario {

	public Root(String login) {
		super(login);
	}

	public void despedidaRoot() {
		System.out.println("Log-off " + super.getLogin());
	}

	public void descricaoRoot() {
		System.out.println("Acesso Total.");
		System.out.println("Você pode excluir todos os usuários");
	}

	public void excluirTodos() {
		System.out.println("Você 'excluiu' todos os usuários");
	}
}

File: Kotlin/TPK3-4/src/Root.kt
class Root(login:String):Usuario(login) {
  fun despedidaRoot() {
    System.out.println("Log-off " + super.getLogin())
  }
  fun descricaoRoot() {
    println("Acesso Total.")
    println("Você pode excluir todos os usuários")
  }
  fun excluirTodos() {
    println("Você 'excluiu' todos os usuários")
  }
}